docs/bdk-cli/installation.md

```diff
@@ -12,12 +12,12 @@ follow their instructions, after which you can test if everything went fine by r
 cargo 1.56.0 (4ed5d137b 2021-10-04)
 ```
 
-If you really don't want to pipe the output of `curl` into `sh`, you can also try using a
+As an alternative to installing the Rust toolchain, you can try using a
 [Docker image](https://hub.docker.com/_/rust) and working inside of it, but that's meant for more
 advanced users and won't be covered in this guide.
 
 ::: tip Note
-At the time of writing, the project requires cargo >= 1.46.0, which is our minimum supported rust version (MSRV) as of July 2021. If you have an older version installed with rustup.rs, you can upgrade it with `rustup update`.
+At the time of writing, the project requires cargo >= 1.56.0, which is our minimum supported rust version (MSRV) as of July 2021. If you have an older version installed with rustup.rs, you can upgrade it with `rustup update`.
```

Comment:
The date needs to be changed too.. It was last updated in May 2022.. https://github.com/bitcoindevkit/bdk-cli/commit/80260e1dd7d3cbd4571e23a125ff1141f7037df2